client-v2/src/main/java/com/clickhouse/client/api/internal/GenericMetric.java

```diff
@@ -0,0 +1,26 @@
+package com.clickhouse.client.api.internal;
+
+import com.clickhouse.client.api.metrics.Metric;
+
+public class GenericMetric implements Metric {

```

Comment:
Metric is intended to be something measurable. String is a constant. It may act as a label.